src/frontend/src/components/SEO.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Helmet from 'react-helmet';
+import useSiteMetadata from '../hooks/use-site-metadata';
+
+function SEO({ description, lang, meta, title }) {
+  const { site } = useSiteMetadata();
```

Comment:
We're getting an error in Zeit with this:

```js
const metaDescription = description || site.siteMetadata.description;
                                                                ^
```

I think we need to change this to:

```js
const { title, description } = useSiteMetadata();
```